images.yaml

```diff
@@ -338,6 +338,15 @@
   tags:
   - sha: 97697df680ea4c7e70c4cb4af5bdc44f7a70b25be7afde70bd921a658e4c62ec
     tag: v0.30.1
+- name: gcr.io/heptio-images/eventrouter
```

Comment:
argh sorting needs to be alphabetical

Reply:
Do I sort by repo or image name?

Reply:
Repo then image - so it should be up there with the other `gcr.io` ones

Reply:
Thanks :) It should be better now :)